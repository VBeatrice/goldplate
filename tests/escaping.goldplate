{
  "command": "cat",
  "arguments": ["-"],
  "stdin": [
    "${VAR}",
    "$${VAR}",
    "$$${VAR}",
    "$$$${VAR}",
    "start ${VAR} end",
    "start $${VAR} end",
    "start $$${VAR} end",
    "start $$$${VAR} end"
  ],
  "environment": {"VAR": "var"},
  "asserts": [
    {"exit_code": 0},
    {"stdout": "${GOLDPLATE_NAME}.stdout"}
  ]
}
