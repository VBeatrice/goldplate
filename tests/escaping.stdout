var
${VAR}
$${VAR}
$$${VAR}
start var end
start ${VAR} end
start $${VAR} end
start $$${VAR} end
